src/main/java/Greetings.java

```diff
@@ -0,0 +1,11 @@
+public class Greetings {
```

Comment:
I'm not sure if makin a class for printing greeting is actually needed. Consider just putting it in the main as constant and using a method to print it.

Reply:
I have refactored it into another class with similar functions. Thank you for the suggestion